cli/modules/modules_test.go

```diff
@@ -1,3 +1,6 @@
+// FIXME: Create new tests https://github.com/tarantool/tt/issues/1039
+//go:build exclude
```

Comment:
This is much worse. We can't break backward compatibility. So old tests should work.

Reply:
According to RFC we don't keep backward compatibility with old module's system.

Reply:
Anyway why does it break here if we add an optional environment variable that is not used in tests?

Reply:
Restore run this test.